frontend/src/product_page/components/ProductCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ProductCard = ({ product }) => {
+  return (
+    <div className="product-card">
+      <p className="product-name">{product.name}</p>
```

Comment:
```suggestion
const ProductCard = ({ product }) => {
  const { name } = product; // 디스트럭쳐링 사용해 깔끔하게 수정~
  
  return (
    <div className="product-card">
      <p className="product-name">{name}</p>
```

destructuring으로 중복 없이 편하게 사용 가능합니다~

cf. [poiemaweb - 객체 디스트럭쳐링](https://poiemaweb.com/es6-destructuring#2-%EA%B0%9D%EC%B2%B4-%EB%94%94%EC%8A%A4%ED%8A%B8%EB%9F%AD%EC%B2%98%EB%A7%81-object-destructuring)

Reply:
수정해두었습니다!